api/v1alpha1/datamoverbackup_types.go

```diff
@@ -32,8 +32,31 @@ type DataMoverBackupStatus struct {
 	// Include references to the volsync CRs and their state as they are
 	// running
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	// Includes source PVC name and size
+	SourcePVCData PVCData `json:"sourcePVCData,omitempty"`
+	// Includes restic repository path
+	ResticRepository string `json:"resticrepository,omitempty"`
```

Comment:
What if the name is something like "DMBResticRepository" or "PVCResticRepository" so that we know it's the repo that we create, not the one given by the user? Just a thought.

Reply:
So basically this is `DMB.Spec.ResticRepository` I think we can rule out `DMBResticRepository` I like `PVCResticRepository` , what abour `SnapshotResticRepository` ?

Reply:
Ah that's a good point. In that case, I think it's fine as is